src/main/java/org/kiwiproject/eureka/junit/EurekaServerExtension.java

```diff
@@ -40,22 +43,37 @@ public EurekaServerExtension(String basePath) {
 
     @Override
     public void beforeAll(ExtensionContext context) {
-        LOG.trace("Creating mock Eureka server");
+        var displayName = context.getDisplayName();
+        LOG.debug("[beforeAll: {}] Initialize testing server.", displayName);
```

Comment:
Suggest changing all these `LOG.debug` calls to be `trace` instead. These appear more like tracing calls to me.

Reply:
(or did you "lift" this logic from the Curator extension? in which case I would have written it, and used DEBUG level 🤣)

Reply:
no, I changed them to debug.  I can change it back